### PR TITLE
set the project modified if track was renamed

### PR DESCRIPTION
**Pull request by BaraMGB**
set the project modified if track was renamed

### CODE REVIEW

**Comment by Umcaruje** [comment]
Tested, works :+1:

File: src/gui/widgets/TrackLabelButton.cpp [hunk 1]
```diff
@@ -107,8 +109,12 @@ void TrackLabelButton::renameFinished()
 	if( !( ConfigManager::inst()->value( "ui", "compacttrackbuttons" ).toInt() ) )
 	{
 		m_renameLineEdit->hide();
-		setText( m_renameLineEdit->text() );
-		m_trackView->getTrack()->setName( m_renameLineEdit->text() );
+		if( m_renameLineEdit->text() != text() )
+		{
+			Engine::getSong()->setModified();
+			setText( m_renameLineEdit->text() );
+			m_trackView->getTrack()->setName( m_renameLineEdit->text() );
```
Comment (zonkmachine) [inline src/gui/widgets/TrackLabelButton.cpp]: I think you should move the ```Engine::getSong()->setModified();``` to here so it's the last in the if(), like on line 90-91^

Comment (BaraMGB) [inline src/gui/widgets/TrackLabelButton.cpp]: sounds logical.

**Comment by BaraMGB** [comment]
Updated.

**Pull request merged**